app.py

```diff
@@ -89,7 +89,11 @@ def predict():
         proba=proba,
         observation=request.data,
     )
-    p.save()
+    try:
+        p.save()
+    except:
+        pass
```

Comment:
This makes a lot of sense! However, catching an error and doing `pass` without returning an error isn't ideal.